concepticondata/conceptlists/Oliveira-2014-517.tsv

```diff
@@ -0,0 +1,518 @@
+Oliveira-2014-517-409	409	debulhar o milho	to thresh the corn					§406
+Oliveira-2014-517-410	410	esp. de palmeira	species of palm					§407
+Oliveira-2014-517-411	411	esp. de lagartixa	species of lizard					§408
+Oliveira-2014-517-412	412	esp. de lagartixa	species of lizard					§409
+Oliveira-2014-517-413	413	esp. de lagarta	species of caterpillar					§410
+Oliveira-2014-517-414	414	esp. de árvore	species of tree					§411
+Oliveira-2014-517-415	415	gordura, óleo	fat, oil		323	FAT (ORGANIC SUBSTANCE)		§412
+Oliveira-2014-517-416	416	velho	old		1229	OLD		§413
+Oliveira-2014-517-417	417	dente, bico (de ave)	tooth, beak of bird				TOOTH//BEAK	§414
+Oliveira-2014-517-418	418	urubu	vulture		1185	VULTURE		§415
+Oliveira-2014-517-419	419	cheirar	to smell		1586	SMELL (PERCEIVE)		§416
+Oliveira-2014-517-420	420	teto (feito de palha)	roof made of straw		769	ROOF		§417
+Oliveira-2014-517-421	421	benefactivo	benefactive					§418
+Oliveira-2014-517-422	422	sufixo de concordancia transitiva	transitive concordance suffix					§419
+Oliveira-2014-517-423	423	assar	to roast		1962	ROAST (SOMETHING)		§420
+Oliveira-2014-517-424	424	agulha	needle		1382	NEEDLE (FOR SEWING)		§421
+Oliveira-2014-517-425	425	coçar, coceira	itch, to itch		148	ITCH		§422
+Oliveira-2014-517-426	426	gordo	fat		1279	FAT (OBESE)		§423
+Oliveira-2014-517-427	427	casa	house		1252	HOUSE		§424
+Oliveira-2014-517-428	428	descascar	to peel		275	PEEL		§425
+Oliveira-2014-517-429	429	esp. de tucano grande	species of large tucan					§426
+Oliveira-2014-517-430	430	pequeno	small		1246	SMALL		§427
+Oliveira-2014-517-431	431	pelar-se, mudar de pele	to peel oneself, to change skin					§428
+Oliveira-2014-517-432	432	seios, leite	breasts, milk		2129	BREAST OR MILK		§429
+Oliveira-2014-517-433	433	verde, nãomaduro	green, unripe		1958	UNRIPE		§430
+Oliveira-2014-517-434	434	mamão	papaya		2445	PAPAYA		§431
+Oliveira-2014-517-435	435	mamão	papaya		2445	PAPAYA		§432
+Oliveira-2014-517-436	436	peito	chest		1592	CHEST		§433
+Oliveira-2014-517-437	437	menina, moça	young woman		77	YOUNG WOMAN		§434
+Oliveira-2014-517-438	438	rato	rat		1490	RAT		§435
+Oliveira-2014-517-439	439	pé	foot		1301	FOOT		§436
+Oliveira-2014-517-440	440	tapiri, casa temporária	tapiri, temporary home					§437
+Oliveira-2014-517-441	441	fígado	liver		1224	LIVER		§438
+Oliveira-2014-517-442	442	amendoim	groundnut		595	GROUNDNUT (PEANUT)		§439
+Oliveira-2014-517-443	443	bochecha	cheek		1730	CHEEK		§440
+Oliveira-2014-517-444	444	bochecha	cheek		1730	CHEEK		§441
+Oliveira-2014-517-445	445	esp. de palmeira	species of palm					§442
+Oliveira-2014-517-446	446	ponte, plataforma	bridge, platform		1011	BRIDGE		§443
+Oliveira-2014-517-447	447	raiz	root		670	ROOT		§444
+Oliveira-2014-517-448	448	pau velho, pau podre	old stick, rotten stick					§445
+Oliveira-2014-517-449	449	rodar	to run		1519	RUN		§446
+Oliveira-2014-517-450	450	roupa	clothes		1895	CLOTHES		§447
+Oliveira-2014-517-451	451	tipo de cesto	type of cedar					§448
```

Comment:
'type of basket'; I  guess that this word talks about a type of handicraft/home-made basket